runtime/syntax/hgcommitDiff.vim

```diff
@@ -0,0 +1,391 @@
+" Vim syntax file
+" Language:	Sapling / Mecurial Diff (context or unified)
```

Comment:
Do we need to support context diff in this file?

Reply:
ngl I don't even know what that means :)